### PR TITLE
Changelog for 3.21.0

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
+	  (ENT-8602)
+	- Stopped loading Apache mod_authz_owner on Enterprise Hubs by default
+	  (ENT-8706)
+	- Stopped loading Apache mod_dbd on Enterprise Hubs by default
+	  (ENT-8609)
+	- Stopped loading mod_authn_file on Enterprise Hubs by default
+	  (ENT-9072)
+	- Stoppped loading mod_authz_dbm on Enterprise Hubs by default
+	  (ENT-8605)
+	- Updated filename conventions for AIX and Solaris packages (ENT-9095)
```
Comment (nickanderson) [inline CHANGELOG.md:47]: Merged ENT-9603 after changelog generated. Adding entry:

```suggestion
	- Updated filename conventions for AIX and Solaris packages (ENT-9095)
	- Fixed detection of location for httpd.pid (ENT-9603)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,51 @@
...
+	  (ENT-9010)
+	- Fixed services_autorun_inputs working independently from services_autorun
+	  (CFE-4017)
+	- Fixed set_line_based() for case when edit_defaults.empty_before_use is true
+	  (ENT-5866)
+	- Improve distributed_cleanup script for testing (ENT-8611)
+	- Made it possible to cfbs add the masterfiles repository directly
+	  (ENT-8915)
+	- Made proc inventory configurable via Augments (CFE-4056)
+	- Make device-tree inventory quieter in containers (#2468) (ENT-9063)
```
Comment (nickanderson) [inline CHANGELOG.md:33]: Redacted github PR number.
```suggestion
	- Make device-tree inventory quieter in containers (ENT-9063)
```

**Review by nickanderson — CHANGES_REQUESTED** [review]
Made some suggestions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,51 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.21.0:
+	- Add inventory for Raspberry Pi and DeviceTree devices (ENT-8628)
```
Comment (olehermanse) [inline CHANGELOG.md:5]:
```suggestion
	- Added inventory for Raspberry Pi and DeviceTree devices (ENT-8628)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,51 @@
...
+	- Added self upgrade knowledge for debian 11 (ENT-9210)
+	- Added ssh in paths.cf so that policy writers can use $(paths.ssh)
+	  (CFE-4037)
+	- Added support for multiple superhubs per feeder (ENT-8936)
+	- Amazon Linux now uses --setopt-exit_on_lock=True in redhat_no_locking_knowledge
+	  (ENT-9057)
+	- Avoided error stopping apache when no pid file exists (ENT-9108)
+	- Disabled explicit setting for SSLCompression for Mission Portal Apache
+	  (ENT-8933)
+	- Fix deleting multiple hosts with distributed cleanup utility
```
Comment (olehermanse) [inline CHANGELOG.md:21]:
```suggestion
	- Fixed deleting multiple hosts with distributed cleanup utility
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,51 @@
...
+	  (ENT-8979)
+	- Fixed directory in which windows agents source packages for upgrade
+	  (ENT-9010)
+	- Fixed services_autorun_inputs working independently from services_autorun
+	  (CFE-4017)
+	- Fixed set_line_based() for case when edit_defaults.empty_before_use is true
+	  (ENT-5866)
+	- Improve distributed_cleanup script for testing (ENT-8611)
```
Comment (olehermanse) [inline CHANGELOG.md:29]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,51 @@
...
+	- Made it possible to cfbs add the masterfiles repository directly
+	  (ENT-8915)
```
Comment (olehermanse) [inline CHANGELOG.md:31]:
```suggestion
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,51 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.21.0:
+	- Add inventory for Raspberry Pi and DeviceTree devices (ENT-8628)
+	- Added check to make sure cf-execd is running after attempting self upgrade on Windows
```
Comment (nickanderson) [inline CHANGELOG.md:6]: Manual changelog entry for https://github.com/cfengine/masterfiles/pull/2554
```suggestion
        - Added policy to enforce proper permissions on Mission Portal ldap directory (ENT-9693)
	- Added check to make sure cf-execd is running after attempting self upgrade on Windows
```

Comment (craigcomstock) [inline CHANGELOG.md:6]: Bad title for https://github.com/cfengine/masterfiles/pull/2555, so here's a better changelog entry:

```suggestion
	- Added check to make sure cf-execd is running after attempting self upgrade on Windows
	- Added exception for settings.ldap.php file for ldap directory permissions (ENT-9697)
```

Comment (craigcomstock) [reply CHANGELOG.md:6]: @nickanderson ^^^

**Review by olehermanse — APPROVED** [review]

**Pull request merged**